WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/logic/ui/UserInactivityNotifier.java

```diff
@@ -2,12 +2,12 @@
 
 import java.util.Date;
 
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent.EventType;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent.EventType;
 
 /**
```

Comment:
i have the feeling we can extract a lot functionality from here to the core, but please keep that in mind for later.